src/main/java/no/nav/veilarbaktivitet/aktiviteter_til_kafka/EndringsType.java

```diff
@@ -24,6 +24,7 @@ public static EndringsType get(AktivitetTransaksjonsType transaksjonsType) {
             case REFERAT_OPPRETTET:
             case REFERAT_ENDRET:
             case REFERAT_PUBLISERT:
+            case FHO_LEST:
```

Comment:
Går det an å kalle denne for FORHAANDSORIENTERING_LEST? Det er litt lettere for en ny person å skjønne enn forkortelsen 😊

Reply:
Tenkte det ble fryktelig langt, men jeg ser poenget :)